### PR TITLE
refactoring/services

### PR DESCRIPTION
**Pull request by Nazar865**
…thods

### CODE REVIEW

File: Corzbank.Api/Controllers/CardsController.cs [hunk 1]
```diff
@@ -45,9 +50,17 @@ public async Task<IActionResult> CreateCard([FromBody] CardModel card)
         }
 
         [HttpDelete("{id}")]
```
Comment (ghost) [inline Corzbank.Api/Controllers/CardsController.cs]: change to http**PUT**, that endpoint will not delete a card, but instead just modify, by setting status to IsClosed or so

File: Corzbank.Services/CardService.cs [hunk 2]
```diff
@@ -36,6 +46,12 @@ public async Task<Card> GetCardById(Guid id)
 
         public async Task<Card> CreateCard(CardModel card)
         {
+            var currentUserEmail = _httpContextAccessor.HttpContext.User.Identity.Name;
+            var currentUser = await _userManager.FindByEmailAsync(currentUserEmail);
+
+            if (await _genericService.FindByCondition(c => c.CardType.Equals(card.CardType) && c.User.Id == currentUser.Id && c.IsActive) != null)
```
Comment (ghost) [inline Corzbank.Services/CardService.cs]: move to a variable, complicated if-statement

**Review by ghost — COMMENTED** [review]
2 minor comments

File: Corzbank.Services/CardService.cs (hunk 2 above)
Comment (Nazar865) [reply Corzbank.Services/CardService.cs]: resolved

File: Corzbank.Api/Controllers/CardsController.cs (hunk 1 above)
Comment (Nazar865) [reply Corzbank.Api/Controllers/CardsController.cs]: resolved

**Review by ghost — APPROVED** [review]

**Pull request merged**